engine/verification/fetcher/consumer.go

```diff
@@ -15,45 +16,44 @@ const (
 	DefaultJobIndex = int64(0)
 )
 
-// ChunkJob converts a Chunk into a Job to be used by job queue
+// ChunkJob converts a chunk locator into a Job to be used by job queue.
 type ChunkJob struct {
-	Chunk *flow.Chunk
+	ChunkLocator *chunks.Locator
 }
 
-// ID converts chunk id into job id, which guarantees uniqueness
+// ID converts chunk locator identifier into job id, which guarantees uniqueness.
 func (j *ChunkJob) ID() module.JobID {
-	return chunkIDToJobID(j.Chunk.ID())
+	return locatorIDToJobID(j.ChunkLocator.ID())
 }
 
-func chunkIDToJobID(chunkID flow.Identifier) module.JobID {
-	return module.JobID(fmt.Sprintf("%v", chunkID))
+func locatorIDToJobID(locatorID flow.Identifier) module.JobID {
+	return module.JobID(fmt.Sprintf("%v", locatorID))
 }
 
-func ChunkToJob(chunk *flow.Chunk) *ChunkJob {
-	return &ChunkJob{Chunk: chunk}
+func ChunkLocatorToJob(locator *chunks.Locator) *ChunkJob {
+	return &ChunkJob{ChunkLocator: locator}
 }
 
-func JobToChunk(job storage.Job) *flow.Chunk {
+func JobToChunkLocator(job storage.Job) *chunks.Locator {
 	chunkjob, _ := job.(*ChunkJob)
-	return chunkjob.Chunk
+	return chunkjob.ChunkLocator
 }
 
-// ChunksJob wraps the storage layer to provide an abstraction for
-// consumers to read jobs
+// ChunksJob wraps the storage layer to provide an abstraction for consumers to read jobs.
 type ChunksJob struct {
```

Comment:
Let's also rename this into `ChunkJobs`

Reply:
Applied in https://github.com/onflow/flow-go/pull/380/commits/298d776fa4ef74e03ea19cbb267af817c393a247.